smithy-aws-rules-engine/src/test/resources/software/amazon/smithy/aws/rulesengine/language/functions/errorfiles/invalid/eventbridge-rules-extra-param.errors

```diff
@@ -0,0 +1,4 @@
+[WARNING] example#FizzBuzz: This shape applies a trait that is unstable: smithy.rules#endpointRuleSet | UnstableTrait
+[WARNING] example#FizzBuzz: This shape applies a trait that is unstable: smithy.rules#endpointTests | UnstableTrait
+[WARNING] example#FizzBuzz: This shape applies a trait that is unstable: smithy.rules#clientContextParams | UnstableTrait
+[DANGER] example#FizzBuzz: Parameter 'useDualStackEndpoint' is never used in test cases | RuleSetParameter.Unused
```

Comment:
This is really confusing - it needs to reference the trait that the issue is under. Otherwise you could could reasonably think it's referring to the endpoint rules trait.